compiler/build.gradle.kts

```diff
@@ -63,6 +63,10 @@ val shadowJar =
         // or we'll get two artifacts that have the same classifier
         archiveClassifier.set("ignored")
       }
+
+      enableRelocation = true
+      relocationPrefix = "dev.zacsweers.metro.compiler.shaded"
```

Comment:
What determines which packages are relocated?

Reply:
The prefix will be added to all packages of depenencies.

Reply:
I do not want to shade all dependencies, only certain dependencies